specs/v1.yaml

```diff
@@ -719,21 +1158,161 @@ components:
           example: "warrenBuffet@example.com"
         note:
           type: string
-          description: An note inputted by the user at submisson 
+          description: An note inputted by the user at submisson
           example: "Please dedicate in memory of grandma"
-        statuses: 
+        statuses:
           type: array
-          items: 
+          items:
             $ref: '#/components/schemas/GrantStatus'
           description: The list of grant statuses for this grant
-    GrantStatus: 
-      type: object 
-      required: 
+    Webhook:
+      type: object
+      description: |-
+        A Webhook object represents an endpoint that will receive requests on certain events
```

Comment:
see above but let's beef up this description. 

e.g.

> A Webhook represents an HTTP endpoint that will receive webhook messages (HTTP requests) on certain events. Webhooks are a way for web applications to communicate with each other in real-time. They enable one application to send automatic notifications or data updates to another application when a specific event occurs.

Reply:
done